fs_s3fs/_s3fs.py

```diff
@@ -353,6 +354,10 @@ def _get_object(self, path, key):
         else:
             return obj
 
+    def _upload_args(self, key):
+        mimetype = mimetypes.guess_type(key)[0] or 'binary/octet-stream'
+        return dict(ContentType=mimetype, **self.upload_args)
```

Comment:
`upload_args` is a free dict right? If the user happened to add `ContentType` to `upload_args` this would raise a TypeError.

Reply:
@willmcgugan oops, good catch! Fixing now